api/src/backend/config/django/base.py

```diff
@@ -207,3 +207,6 @@
 
 
 TESTING = False
+
+# Disable RBAC during tests/demos
+DISABLE_RBAC = False
```

Comment:
```suggestion
```

Reply:
For the moment we are going to keep this flag since its deletion implies refactoring a great part of the API unit tests, we will write down this task to be done in the near future